ingeniamotion/communication.py

```diff
@@ -827,6 +828,9 @@ def disconnect(self, servo: str = DEFAULT_SERVO) -> None:
         del self.mc.servos[servo]
         net_name = self.mc.servo_net.pop(servo)
         servo_count = list(self.mc.servo_net.values()).count(net_name)
+        # TODO: Remove once INGK-912 is resolved
+        with contextlib.suppress(NotImplementedError):
```

Comment:
Why is this dependant on the release?
Create a task to do this or add it to the issue, otherwise we might forget

Reply:
Because right now the fsoe_master library is only being installed on one machine (the one that is able to compile it).